_source/_posts/2021-10-16-full-stack-java.md

```diff
@@ -0,0 +1,693 @@
+---
+layout: blog_post
+title: "Full Stack Java with React, Spring Boot, and JHipster"
+author: matt-raible
+by: advocate
+communities: [java,javascript]
+description: "This tutorial shows you how to be a full stack Java developer with React and Spring Boot."
+tags: [java, full-stack, react, spring-boot, jhipster]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+---
+
+If you search for "Full Stack Java" on the internet, you'll likely find a lot of recruiting, courses, and jobs. Being a full stack developer can be exciting because you're learning and using a bunch of technologies. It also pays pretty well. Today, I'm going to show you how you can be a full stack Java developer with Spring Boot, React, and JHipster.
```

Comment:
I have doubts related to the second sentence, "Being a full stack developer can be exciting because you're learning and using a bunch of technologies". Those points that you mentioned relates not only to a full stack dev, but also dev focused on backend or frontend only ;) How about we will be here a little bit more precise.  I think being a full stack can be exciting because you can create your own app that consists of backend and frontend all by your own. There is some algorithm work (backend) as well as styling etc. What do you think about those points?